experimental/terraform/modules/fourkeys-bigquery/main.tf

```diff
@@ -0,0 +1,132 @@
+
+resource "google_project_service" "bigquery" {
+  project                    = var.project_id
+  service                    = "bigquery.googleapis.com"
+  disable_dependent_services = true
+
+}
+
+resource "google_project_service" "bigquery_storage" {
```

Comment:
Hrm - do we use this?  It's unfamiliar to me.  @davidstanke

Reply:
Hmm, yeah, we haven't used it before. @rogerthatdev what's it for?

Reply:
Bigquery Storage is automatically enabled with the Bigquery API. Including it or excluding it shouldn't make a difference.  I think I included it after I checked what API's were enabled when I ran through the setup steps.  Including it or excluding it shouldn't make a difference.